dGame/dBehaviors/AreaOfEffectBehavior.cpp

```diff
@@ -7,7 +7,6 @@
 #include "Logger.h"
 #include "BehaviorBranchContext.h"
 #include "BehaviorContext.h"
-#include "RebuildComponent.h"
```

Comment:
To hold us to non-functional changes/renaming truly being non-functional, I would like to request that this include be added back and just renamed for consistency.